### PR TITLE
Dia landing page

### PR DESCRIPTION
**Pull request by lanebx**
[DEMO LINK](https://lanebx.github.io/layout_dia/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -1,13 +1,162 @@
...
+        <div class="header__logo-box">
+          <div class="header__logo"></div>
+          <div class="header__burger"></div>
+        </div>
+        
+        <div class="header__nav">
+          <ul class="header__list">
+            <li class="header__item">
+              <a href="#about-us" class="header__link">About us</a>
+            </li>
```
Comment (volodymyr-soltys97) [inline src/index.html:26]: Пам'ятай про стиль написання коду, перевір це по всій сторінці
```suggestion
            <li class="header__item">
              <a href="#about-us" class="header__link">
                About us
              </a>
            </li>
```

File: src/index.html [hunk 2]
```diff
@@ -1,13 +1,162 @@
...
+            </div>
+          </div>
+
+          <div class="expertise__item">
+            <img src="images/what-we-do/what-we-do-2.jpg" class="expertise__img">
+            <div class="expertise__item-title">
+              Communication
+            </div>
+            <div class="expertise__text">
+              We strive to create communications that can increase media performance. We use everything — words, meanings, stories, art, movies.            </div>
```
Comment (volodymyr-soltys97) [inline src/index.html:127]:
```suggestion
            <div class="expertise__text">
              We strive to create communications that can increase media performance. We use everything — words, meanings, stories, art, movies. 
            </div>
```

File: src/index.html [hunk 3]
```diff
@@ -1,13 +1,162 @@
...
+              Strategy
+            </div>
+            <div class="expertise__text">
+              We create business growth strategies, from the moment of its birth to the achievement of the necessary business indicators.            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+
+    <div class="services" id="ervices">
```
Comment (volodymyr-soltys97) [inline src/index.html:142]: Можливо id="services"?

File: src/styles/main.scss [hunk 4]
```diff
@@ -1,7 +1,372 @@
...
+  &__hair-us {
+    text-transform: uppercase;
+  }
+
+  &__hair-us::after {
+    display: block;
+    content: "";
+    height: 2px;
+    margin-top: 8px;
+    background: #2060F6;
```
Comment (volodymyr-soltys97) [inline src/styles/main.scss:122]: background-color можливо?

File: src/styles/main.scss [hunk 5]
```diff
@@ -1,7 +1,372 @@
...
+  &__subtitle {
+    margin-bottom: 83px;
+    min-width: 280px;
+    max-width: 450px;
+    font-family: "Open Sans";
+    font-style: normal;
+    font-weight: normal;
+    font-size: 18px;
+    line-height: 150%;
+    color: #FFFFFF;
```
Comment (volodymyr-soltys97) [inline src/styles/main.scss:152]: Я думаю варто створити змінні для кольорів які ти використовуєш більше одного разу

**Review by volodymyr-soltys97 — CHANGES_REQUESTED** [review]
В цілому гарна робота, тільки варто розділити стилі на різні файли, дуже багато рядків коду зараз. Також потрібно додати іконку для сторінки у вкладці. Також чомусь не відкривається меню на мобільній версії і немає кнопки відкриття, + переключатель для слайдеру позиціонується трохи не там де потрібно у мобільній версії, ось скрін 
image

File: src/index.html (hunk 3 above)
Comment (lanebx) [reply src/index.html]: corrected

File: src/styles/main.scss (hunk 4 above)
Comment (lanebx) [reply src/styles/main.scss]: corrected

**Review by aionova — CHANGES_REQUESTED** [review]
Screenshot 2021-11-02 at 16 11 51

- hire us should be a clickable link.
- all the buttons and clickable elements should have hover-effect, transition and cursor: pointer
- all buttons should do something, lead you to another part of the website.
- on small screens I see some weird white thing)
Screenshot 2021-11-02 at 16 16 16
Keep working!)

**Review by ihor-jpeg — CHANGES_REQUESTED** [review]
Good stuff! 🔥  There are a few things to continue working on

- these buttons should have `cursor: pointer` and, optionally, some hover effects
Screenshot 2021-11-04 at 11 45 37

- add hover effects to links
Screenshot 2021-11-04 at 11 45 45
Screenshot 2021-11-04 at 11 46 50

- it might be just me, but letter-spacing here is a bit too much
Screenshot 2021-11-04 at 11 46 31

- the button has a strange border, which disappears on hover
Screenshot 2021-11-04 at 11 47 00

- and lastly, here is a challenge: can you spot the similarities between these two images? 😆 
Screenshot 2021-11-04 at 11 46 53
Screenshot 2021-11-04 at 11 55 49

**Review by ihor-jpeg — CHANGES_REQUESTED** [review]
Your work gets better and better!

- the menu is open by default, but it should not be
Screenshot 2021-11-04 at 17 52 05
Screenshot 2021-11-04 at 17 52 12

- there is transition duration on header links, but none on button hover
Screenshot 2021-11-04 at 17 52 27

- when i click on the button, the background changes to while
Screenshot 2021-11-04 at 17 52 22

- in some cases 100vh will throw off the layout, for example if a user has a tall screen (this is just my opinion and it's up to you)
Screenshot 2021-11-04 at 17 54 00

**Review by AlinaMatuschak — CHANGES_REQUESTED** [review]
Nice work. But we can better 💪

- Can you render mobile menu on all viewport width&height?
Screenshot 2021-11-08 at 20 54 55
- on click"learn more" and "apply" buttons you can lead to the contact block
- small space after button
Screenshot 2021-11-08 at 20 58 46
- add hover for these blocks
Screenshot 2021-11-08 at 20 59 36

**Review by AlinaMatuschak — CHANGES_REQUESTED** [review]
- fix autocomplete
Screenshot 2021-11-09 at 12 52 31
 
- hover for logo
- on click learn more lead to services block
- on click apply lead to form block
- bug with footer
Screenshot 2021-11-09 at 14 25 59

**Review by AlinaMatuschak — CHANGES_REQUESTED** [review]
Are you sure that you redeployed?

**Comment by lanebx** [comment]
> Are you sure that you redeployed?

you're right,
already deployed

**Review by aionova — CHANGES_REQUESTED** [review]
- these Learn more buttons should also do something
Screenshot 2021-11-11 at 12 17 35
- this button should lead you to the form
Screenshot 2021-11-11 at 12 18 38
 - when I use autocomplete, the input doesn't look good
Screenshot 2021-11-11 at 12 19 11
- Location of burger-menu icon is not obvious at all. It should be in the right corner
Screenshot 2021-11-11 at 12 21 50
- burger-menu should close when you press on one of the items in the menu
- you can add "close" button to the menu.
- make the block paddings and margins smaller on small screens
Screenshot 2021-11-11 at 12 33 21

**Review by aionova — APPROVED** [review]
GJ!

**Pull request closed without merging**